browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -312,37 +286,10 @@ BraveBrowserView::BraveBrowserView(std::unique_ptr<Browser> browser)
       browser_->is_type_normal()) {
     SplitViewBrowserData::CreateForBrowser(browser_.get());
```

Comment:
Can we create this inside `SplitView`? As a follow-up we should look at refactoring this per https://chromium.googlesource.com/chromium/src/+/master/docs/chrome_browser_design_principles.md

Reply:
Moved this creation to `SplitView`.